site/source/user-manual/flashing/flashing-firmware.rst

```diff
@@ -15,9 +15,12 @@ The source code can be viewed on Purism's `firmware git repo <https://source.pur
 
     .. note:: You'll need a monitor and keyboard plugged into your Librem Mini for this operation
 
-#. Turn on the Librem Mini while pressing the ``ESC`` key on the keyboard repeatedly, until you see the PureBoot Basic Boot Menu appear.
+#. Turn on the Librem Mini while pressing the ``ESC`` key on the keyboard repeatedly while you see this screen:
```

Comment:
This (and the next change that adds the PUREBOOT screen screenshot) may not be necessary as it's possible they confuse a person more than they help, given that if the person is flashing because Purism didn't do their job and the firmware doesn't exist, this screen also won't exist.  Thoughts?